conda-forge.yml

```diff
@@ -2,4 +2,10 @@ channel_priority: strict
 azure:
   settings_win:
     pool:
-      vmImage: windows-2019
\ No newline at end of file
+      vmImage: windows-2019
+build_platform:
+  osx_arm64: osx_64
+os_version:
+  linux_64: cos7
+test: native_and_emulated
+      
```

Comment:
Do not edit any files outside your recipe directory on staged-recipes. If you want to add keys to conda-forge.yml. Add this file into your recipe directory, and the bot will move it to the correct place when your feedstock is created.